README.md

```diff
@@ -53,10 +53,44 @@ We are using continuous integration software applied to the supplied test datase
 
 ### Folder Structure
 
+Our folder structure is designed to separate each analysis into its own set of notebooks that are independent of other analyses.
+Within the analyses directory, create a folder for your analysis.
+Choose a name that is unique from other analyses and somewhat detailed.
+For example, instead of `gene-expression`, choose `gene-expression-clustering` if you are clustering samples by their gene expression values.
+You should assume that any data files that you need to read live in the `../../data` directory and that their file names match what the `data_downloader.sh` script produces.
+These files should be read in at their relative path, so that we can re-run analyses if the underlying data change.
+
+```
+OpenPBTA-analysis
+├── CONTRIBUTING.md
+├── README.md
+├── analyses
+│   ├── existing-analysis-1
+│   ├── existing-analysis-2
+│   └── new-analysis
+│       ├── 00.run_all.sh
+│       ├── 01.preprocess-data.Rmd
+│       ├── 02.run-analyses.Rmd
+│       └── 03.make-figures.Rmd
+└── data
+└── results
+│   └── new-analysis
+│       ├── figure1.pdf
```

Comment:
Does this mean we generally want things to be in PDF form when possible or is this not actually a preference you're meaning to imply by this?

Reply:
Updated to say:
> The author has produced their output figures as `.pdf` files.
> We have a preference for vector graphics as PDF files, though other forms of vector graphics are also appropriate.